### ISSUE TITLE
njs module exports

### ISSUE THREAD

**Issue by oshatrk** [Microsoft/TypeScript#14610]
**TypeScript Version:**  2.1.1 / nightly (2.2.0-dev.201xxxxx)

**Code**

```ts
// A *self-contained* demonstration of the problem follows...

```

**Expected behavior:**

**Actual behavior:**

**Comment by DanielRosenwasser** [comment]
Please reopen with a complete example if you meant to post something.

**Comment by oshatrk** [comment]
Don't worry this issue was created accidentally. I imagined what happend:

While I was looking to one of issues in this repository I've tried to google `commonjs module exports`.
But didn't saw I've not focused a browser's location bar. 
So `c` cause to create new github issue (as it's one of [Github shortcut](https://help.github.com/articles/using-keyboard-shortcuts/#issue-and-pull-request-lists)).
And while I typed `ommo` the New issue pages was loaded and as its input tag has an autofocus attribute so `njs module exports` was typed as title.
Then `Enter` in this Github's input is binded to submit new issue even there is an empty issue description.

Sorry for that. Next time I'll be more accurate.
